gatsby_fp/src/components/fields.js

```diff
@@ -0,0 +1,89 @@
+import React from "react"
+import GenInfoTable from "./fieldComponents/genInfoTable"
```

Comment:
As a matter of convention, filenames for components should be PascalCase. Your IDE should automatically fix imports if you're using VSCode when you rename them, but it pays to double-check to confirm.